test/test-get-currentexe.c

```diff
@@ -82,5 +82,27 @@ TEST_IMPL(get_currentexe) {
   ASSERT(buffer[0] != '\0');
   ASSERT(buffer[1] == '\0');
 
+  /* Verify uv_exepath is not affected by uv_set_process_title(). */
+  r = uv_set_process_title("foobar");
+  ASSERT(r == 0);
+  size = sizeof(buffer) / sizeof(buffer[0]);
+  r = uv_exepath(buffer, &size);
+  ASSERT(!r);
+
+  /* uv_exepath can return an absolute path on darwin, so if the test runner
+   * was run with a relative prefix of "./", we need to strip that prefix off
+   * executable_path or we'll fail. */
+  if (executable_path[0] == '.' && executable_path[1] == '/') {
+    path = executable_path + 2;
+  } else {
+    path = executable_path;
+  }
```

Comment:
No braces around single-line consequents/alternates. Also, a slightly more succinct way of writing it:
```c
path = executable_path;
if (path[0] == '.' && path[1] == '/')
  path += 2;
```